pkg/nsx/services/common/store.go

```diff
@@ -192,19 +192,25 @@ func (service *Service) PopulateResourcetoStore(wg *sync.WaitGroup, fatalErrors
 
 // InitializeCommonStore is the common method used by InitializeResourceStore and InitializeVPCResourceStore
 func (service *Service) InitializeCommonStore(wg *sync.WaitGroup, fatalErrors chan error, org string, project string, resourceTypeValue string, tags []model.Tag, store Store) {
-	tagScopeClusterKey := strings.Replace(TagScopeCluster, "/", "\\/", -1)
-	tagScopeClusterValue := strings.Replace(service.NSXClient.NsxConfig.Cluster, ":", "\\:", -1)
-	tagParam := fmt.Sprintf("tags.scope:%s AND tags.tag:%s", tagScopeClusterKey, tagScopeClusterValue)
+	var tagParams []string
+	// Check for specific tag scopes
+	if !containsTagScope(tags, TagScopeCluster, TagScopeNCPCluster) {
+		tagParams = append(tagParams, formatTagParamScope("tags.scope", TagScopeCluster))
```

Comment:
Why it checks both cluster and ncp cluster tag, then only append cluster tage?

Reply:
To tolerate previous scenarios, if not includeNCP, we should only search nsx side resources.